neural_compressor/model/onnx_model.py

```diff
@@ -423,3 +427,15 @@ def get_nodes_chain(self, start_node, stop_node, result_chain=[]):
                 start_node.append(parent.name)
 
         return result_chain
+
+    def export(self, save_path, conf):
+        from neural_compressor.experimental.export import onnx_qlinear_to_qdq
+        add_nodes, remove_nodes, inits = onnx_qlinear_to_qdq(self._model,
+                                         self._input_name_to_nodes)
+        self.add_nodes(add_nodes)
```

Comment:
I don't understand this part. export() should support all conversions, such as QDQ to Qlinear, Qlinear to QDQ. why lines 433 directly invokes Qlinear2QDQ? I didn't see you parse conf file. what will happen if the input model is QDQ?

Reply:
Currently we only target Qlinear2QDQ for 2.0, add judgement for the conf